src/errors/test-run/templates.js

```diff
@@ -223,7 +223,9 @@ export default {
     `),
 
     [TYPE.externalAssertionLibraryError]: err => markup(err, `
-        ${escapeHtml(err.errMsg)} ${err.getCallsiteMarkup({ stackOnly: true })}
+        ${escapeHtml(err.errMsg)}
+
+        ${err.getCallsiteMarkup()}
```

Comment:
BTW, seems like we don't need `stackOnly` option anymore (need to check this), so we can remove it now from `getCallsiteMarkup`